docs/static/releasenotes.asciidoc

```diff
@@ -70,28 +70,18 @@ chain TLS certificate verification.
 [[plugins-7-15-2]]
 ==== Plugins
 
-*Geoip Filter - 7.2.4*
-
-* Fix: update to Gradle 7 https://github.com/logstash-plugins/logstash-filter-geoip/pull/191[#191]
-* [DOC] Clarify CC licensed database indefinite use condition and air-gapped environment https://github.com/logstash-plugins/logstash-filter-geoip/pull/192[#192]
-
 *Beats Input - 6.2.1*
 
-* Fix: LS failing with `ssl_peer_metadata => true` https://github.com/logstash-plugins/logstash-input-beats/pull/431[#431]
-* [DOC] described `executor_threads` configuration parameter https://github.com/logstash-plugins/logstash-input-beats/pull/421[#421]
```

Comment:
Same comment here. We generally list more lower level changes for plugins that we do for Logstash core.  If someone isn't interested in the plugin, they can skip that section. If they are interested in that plugin, they can decide if they're interested enough for follow links and/or upgrade.